docs/guides/building/windows.md

```diff
@@ -24,40 +24,104 @@ tauri build --target i686-pc-windows-msvc
 By default Rust only installs toolchains for your machine's target, so you need to install the 32-bit Windows toolchain first: `rustup target add i686-pc-windows-msvc`. You can get a full list of Rust targets by running `rustup target list`.
 :::
 
-## Using a Fixed Version of the Webview2 Runtime
+## Supporting Windows 7
 
-By default, the Tauri installer downloads and installs the Webview2 Runtime if it is not already installed (on Windows 11, WebView2 is preinstalled).
+By default the Microsoft Installer does not work on Windows 7 because it needs to download the Webview2 bootstrapper if the runtime is not installed, which might fail if TLS 1.2 is not enabled in the operating system. Tauri includes an option to embed the Webview2 bootstrapper, see the [section below](#embedding-the-webview2-bootstrapper).
+
+Additionally, to use the Notification API in Windows 7, you need to enable the `windows7-compat` Cargo feature:
+
+```toml title="Cargo.toml"
+[dependencies]
+tauri = { version = "1", features = [ "windows7-compat" ] }
+```
+
+## Distributing a 32-bit executable
+
+The Tauri CLI compiles your executable using your machine's architecture by default. You can target a different one using the `target` argument. For instance, to generate an installer for the x86 architecture, you can execute `tauri build --target i686-pc-windows-msvc` after installing the _i686-pc-windows-msvc_ target running `rustup target add i686-pc-windows-msvc`. You can see the list of available targets running `rustup target list`.
+
+## Using alternative Webview2 installation options
+
+The Windows Installer by default downloads the Webview2 bootstapper and executes it if the runtime is not installed. Alternatively, you can embed the bootstrapper, embed the offline installer or use a fixed Webview2 runtime version. See the following table for a comparison between these methods:
+
+| Install method        | Requires internet connection | Added installer size |
+| :-------------------- | :--------------------------- | :------------------- |
+| Download bootstrapper | :heavy_check_mark:           | 0MB                  |
+| Embed bootstrapper    | :heavy_check_mark:           | ~1.8MB               |
+| Embed installer       |                              | ~127MB               |
+| Fixed version         |                              | ~180MB               |
 
 :::note
-You can remove the Webview2 Runtime download check from the installer by setting [`tauri.bundle.windows.wix.skipWebviewInstall`] to `true`.
+On Windows 10 (since June 27, 2022) and Windows 11, the Webview2 runtime is distributed as part of the operating system.
+:::
 
-**Your application WILL NOT work if the user does not have the runtime installed with this option selected.**
+:::note
```

Comment:
Does it make sense to move this note down below the json code in the below section? A bit weird to have 2 notes back to back